### ISSUE TITLE
dunplicate default in webpack output 'commonjs2' style libs

### ISSUE THREAD

**Issue by githoniel** [systemjs/systemjs#1649]
I use webpack to output es6 js  libaray as 
```
libraryTarget: 'commonjs2'
```

the use SystemJS to load them 

in 0.20.9 , It's ok I can get the object by default property
```
SystemJS.import('path').then( m => m.default.init())
```
but in 0.20.11, I must use as

```
SystemJS.import('path').then( m => m.default.default.init())
```
debugger show as below
![qq 20170406165051](https://cloud.githubusercontent.com/assets/17848159/24745894/de841856-1ae9-11e7-9248-3f5cf43c2fea.png)

**Comment by guybedford** [comment]
Unfortunately this is correct, yes.

You can simplify to just `m.default.init` by adding the `esModule: true` metadata property to the module before importing:

```js
System.config({
  meta: {
    path: { esModule: true }
  }
});
```

**Comment by githoniel** [comment]
I tried like below  but no use 
```javascript
   return fetch(mapJsonLocation)
        .then(response => response.json())
        .then((asset) => {
            const map = {}
            const meta = {}
            if (asset.metadata) {
                delete asset.metadata
            }
            Object.keys(asset).forEach((key) => {
                const aliasKey = `${packageName}/${key}`
                const aliasPath = `${prefixPath}/${asset[key].js}`
                map[aliasKey] = aliasPath
                meta[aliasKey] = {
                    esModule: true
                }
            })
            SystemJS.config({
                meta,
                map
            })
```

and sorry I don't understand why double default is correct, in old version systemjs it is only one default

**Comment by guybedford** [comment]
Right, but the module is located by export as `exports.default` where `exports.default.__esModule` exists, not `exports.__esModule` right? If this isn't the case let me know though.

I'd suggest working out the right webpack config to ensure it isn't set as a default export, otherwise perhaps a wrapper can work in SystemJS along the lines of `module.exports = require('x').default`.

**Comment by githoniel** [comment]
it output like below, bizType is my export default object
![qq 20170406230943](https://cloud.githubusercontent.com/assets/17848159/24761514/28c18974-1b1e-11e7-9d7f-c1003ecc1009.jpg)

I roll back to 0.20.9, and  it's all right now, so i don't think it's webpack config's problem. 
this is one of my  webpack output libary
```js
module.exports =
webpackJsonp1491484298553([71],{

/***/ 170:
/***/ (function(module, __webpack_exports__, __webpack_require__) {

"use strict";
Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
/* harmony export (binding) */ __webpack_require__.d(__webpack_exports__, "__useDefault", function() { return __useDefault; });
/* harmony default export */ __webpack_exports__["default"] = {
    bizType: {
        'revoke': 1,
        'refund': 2,
        'balance': 3,
        'others': 99
    }
};

var __useDefault = true;

/***/ }),

/***/ 327:
/***/ (function(module, exports, __webpack_require__) {

module.exports = __webpack_require__(170);


/***/ })

},[327]);
//# sourceMappingURL=bundle.71.js.map
```

**Comment by guybedford** [comment]
I've been working on a fix for this coming in the next patch release. It should help here.

**Comment by guybedford** [comment]
Released in 0.20.13.
